sdk/src/main/java/com/mapbox/maps/ThreadChecker.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.maps
+
+import android.os.Looper
+import android.util.Log
+import com.mapbox.common.MapboxSDKCommon
+
+internal object ThreadChecker {
+
+  // Captures if the consumer application is running in debug
+  private var debug = false
+  private const val TAG = "mbgl-ThreadChecker"
```

Comment:
with new logging no need to add `mbgl-`, just `ThreadChecker` will be fine and will result in:
```
I/Mapbox: [maps-android\Mbgl-Renderer]: message
```